analytics/upress_reporting/models/data/interaction_event.py

```diff
@@ -11,7 +11,7 @@ class InteractionType(Enum):
 class UsageType(Enum):
     FULL_ACCESS = "Full Access"
     LIMITED_ACCESS = "Limited Access"
-    VIEW_ONLY_NO_DOWNLOAD_ACCESS = "View Only / No Download Access"
+    VIEW_ACCESS = "View Only / No Download Access"
```

Comment:
Should the string here be `View Access` or `View`?

Reply:
I did `View Access` since it matches the format of the other access types, but I can switch it if need be!